docs/rabbitmq.config.example

```diff
@@ -319,9 +346,19 @@
    %%
    %% {collect_statistics_interval, 5000},
 
-   %% Explicitly enable/disable hipe compilation.
+   %% Enables vhosts tracing.
+   %%
+   %% {trace_vhosts, []},
+
+   %% Explicitly enable/disable HiPE compilation.
    %%
-   %% {hipe_compile, true},
+   %% {hipe_compile, false},
+
+   %% Number of delegate processes to use for intra-cluster communication.
+   %% On a machine which has a very large number of cores and is also part of a cluster,
```

Comment:
How large is "very large"? I'd say "more than 16 cores and plenty of bandwidth".